### ISSUE TITLE
Remove terminal colour control characters from log files on linux

### ISSUE THREAD

**Issue by LulaSvob** [trexminer/T-Rex#764]
I am using a log file which I move to a web server to be able to track miner performance remotely without the need to login. I do this successfully on a windows machine with several cards but on my Ubuntu machine I get some very weird characters in the log:

```
20211023 11:55:34 GPU #0: intensity 21.3
20211023 11:55:34 Using protocol: stratum1.
20211023 11:55:34 Authorizing...
20211023 11:55:34 Authorized successfully.
20211023 11:55:34 ethash epoch: 449, block: 13472815, diff: 8.73 G
20211023 11:55:37 GPU #0: generating DAG 4.51 GB for epoch 449 ...
20211023 11:55:46 [0;36mGPU #0: DAG generated [crc: 574540de[0m[0;36m, time: 8826 ms], memory left: 2.02 GB
[0m20211023 11:55:46 GPU #0: using kernel #4
20211023 11:55:48 GPU #0: target hashrate for unlocker - 43.68 MH/s
20211023 11:55:50 [32m[ OK ][0m 1/1 - 44.25 MH/s, 47ms ... GPU #0
[0m20211023 11:55:54 ethash epoch: 449, block: 13472816, diff: 8.73 G
20211023 11:56:01 ethash epoch: 449, block: 13472817, diff: 8.73 G
20211023 11:56:04 ethash epoch: 449, block: 13472818, diff: 8.73 G
20211023 11:56:07 ethash epoch: 449, block: 13472819, diff: 8.73 G
20211023 11:56:33 ethash epoch: 449, block: 13472820, diff: 8.73 G
20211023 11:56:49 ethash epoch: 449, block: 13472821, diff: 8.73 G
20211023 11:57:03 ethash epoch: 449, block: 13472822, diff: 8.73 G

-------------20211023 11:57:05 -------------
Mining at eth.2miners.com:2020, diff: 8.73 G
[0;97mGPU #0: [0m[0;97mRTX 3070 - 43.98 MH/s, [LHR 71<>] [[0m[0;97mT:[0m[32m59[0m[0;97mC, [0m[0;97mP:149W, [0m[0;97mF:65%, [0m[0;97mE:297kH/W[0m[0;97m][0m[0;97m, 1/1 R:[0m[0;97m0[0m[0;97m%[0m
[0mShares/min: 1 (Avg. 4)
Uptime: 1 min 30 secs | Algo: ethash | T-Rex v0.24.2
WD: 1 min 31 secs, shares: 1/1 

20211023 11:57:35 ethash epoch: 449, block: 13472823, diff: 8.73 G

-------------20211023 11:57:35 -------------
Mining at eth.2miners.com:2020, diff: 8.73 G
[0;97mGPU #0: [0m[0;97mRTX 3070 - 43.80 MH/s, [LHR 71<>] [[0m[0;97mT:[0m[32m60[0m[0;97mC, [0m[0;97mP:149W, [0m[0;97mF:65%, [0m[0;97mE:296kH/W[0m[0;97m][0m[0;97m, 1/1 R:[0m[0;97m0[0m[0;97m%[0m
[0mShares/min: 1 (Avg. 4)
Uptime: 2 mins | Algo: ethash | T-Rex v0.24.2
WD: 2 mins 1 sec, shares: 1/1 

20211023 11:57:50 ethash epoch: 449, block: 13472824, diff: 8.73 G
20211023 11:57:51 ethash epoch: 449, block: 13472825, diff: 8.73 G
20211023 11:57:59 ethash epoch: 449, block: 13472826, diff: 8.73 G
```
Setup is practically the same on windows and linux and I have no idea where these chars like `[0;97m` and `[0m[32m` are coming from - any ideas? 

Messages in terminal are just fine. Those appear only in the file that I pass to the `--log-path` parameter. 

Here is how the files look when opened in VS Code - left is from Ubuntu and the right is from Windows
![image](https://user-images.githubusercontent.com/32610546/138552783-d3200c3c-1785-400f-94fd-96b19d792ba4.png)

And here is how the log looks like in the terminal i.e. no issues and all is fine:
![image](https://user-images.githubusercontent.com/32610546/138552679-419bb12b-3931-42ae-a1b5-9fbffe96f5a7.png)

I tried opening the file with different editors and with different encodings but to no avail. 
Have anyone else faced the same issue and what could a potential fix be in this case?

**Comment by trexminer** [comment]
Those are special characters used to control terminal colours for certain blocks of text, like temperatures and so on.
Getting rid of them in the log files is on our TODO list, although not a high priority at the moment.

**Comment by trexminer** [comment]
Fixed in 0.24.6
